### PR TITLE
Adds Verification Commands.

### PR DESCRIPTION
**Pull request by lajay-lawrence**
#### What type of PR is this?
/kind bug

#### What this PR does / why we need it:
Adds verification commands to 'Verify-workerServices' to ensure nodes are connected to API server, successfully join a cluster and are in ready state.

#### Which issue(s) this PR fixes:
b/227506769

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cluster/gce/windows/k8s-node-setup.psm1 [hunk 1]
```diff
@@ -1118,12 +1118,33 @@ function WaitFor_KubeletAndKubeProxyReady {
...
-# TODO(pjh): run more verification commands.
+# Runs additional verification commands to ensure node succesfully joined cluster
+# and that it connects to the API Server.
 function Verify-WorkerServices {
-  Log-Output ("kubectl get nodes:`n" +
-              $(& "${env:NODE_DIR}\kubectl.exe" get nodes | Out-String))
+  $timeout = 12
+  $retries = 0
+  
+  Write-Host("Testing node connection to API server...")
```
Comment (ibabou) [inline cluster/gce/windows/k8s-node-setup.psm1:1127]: Let's use Log-Output?

File: cluster/gce/windows/k8s-node-setup.psm1 [hunk 2]
```diff
@@ -1118,12 +1118,33 @@ function WaitFor_KubeletAndKubeProxyReady {
 }
 
 # Runs 'kubectl get nodes'.
-# TODO(pjh): run more verification commands.
+# Runs additional verification commands to ensure node succesfully joined cluster
+# and that it connects to the API Server.
 function Verify-WorkerServices {
-  Log-Output ("kubectl get nodes:`n" +
-              $(& "${env:NODE_DIR}\kubectl.exe" get nodes | Out-String))
+  $timeout = 12
```
Comment (ibabou) [inline cluster/gce/windows/k8s-node-setup.psm1:1124]: nit: keep a variable with sleep period as well, like $retryDelayInSeconds. to be easier to figure the total retry time.

**Review by ibabou — APPROVED** [review]

File: cluster/gce/windows/k8s-node-setup.psm1 (hunk 1 above)
Comment (lajay-lawrence) [reply cluster/gce/windows/k8s-node-setup.psm1:1127]: Yes! I was debating that in my mind, thank you

File: cluster/gce/windows/k8s-node-setup.psm1 (hunk 2 above)
Comment (lajay-lawrence) [reply cluster/gce/windows/k8s-node-setup.psm1:1124]: will do, thank you

**Review by ibabou — APPROVED** [review]
Thanks Lajay!

**Review by ibabou — APPROVED** [review]
Can you squash the commits into one? with "git rebase -i"?

**Pull request merged**